c_cpp_guide.md

```diff
@@ -0,0 +1,111 @@
+# Compiling C/C++ to Ewasm
+
+First an introduction, then a basic step-by-step guide, then advanced things. Warning: the Ewasm spec and tools below are subject to change.
```

Comment:
> Warning: the Ewasm spec and tools below are subject to change.

I understand why you'd add this caveat here, but I don't find it constructive or helpful on its own, i.e., it just makes the reader worry that the instructions aren't going to work. Consider making this more constructive by saying something like, "Every effort is made to keep this document up to date, but if you notice anything wrong please feel free to submit a PR or an issue to report and/or fix it."

Reply:
Agreed. It is understood that things are subject to change. I completely removed it so that the guide is more concise.